lesson19/news-article.json

```diff
@@ -0,0 +1,312 @@
+{
+  "data": [
+    {
+      "id": "624ea99d26476db2ff6fc08c",
+      "isActive": false,
+      "picture": "img/cat-news.png",
+      "categories": "News",
```

Comment:
Q.
It is in the plural, do you anticipate that there will be more categories?

Reply:
Thank you for your Comment!
The category is singular because it is not an array. Therefore, the singular form was appropriate.
I fixed.
https://github.com/nakagawamai/js-lesson/pull/21/commits/281e6769d1ec05bc2c8609e8ca35837f7fba840c